mod_reforged/hooks/skills/racial/golem_racial.nut

```diff
@@ -77,6 +77,12 @@
 				type = "text",
 				icon = "ui/icons/morale.png",
 				text = ::Reforged.Mod.Tooltips.parseString("Not affected by [Morale|Concept.Morale]")
+			},
+			{
+				id = 29,
+				type = "text",
+				icon = "ui/icons/special.png",
+				text = "Immune to Headshots"
```

Comment:
I think this should read "Cannot receive hits to the head". Because `Immune to Headshots` seems to imply that you receive a hit to the head but are immune to extra damage from it. `Immune to Headshots` should probably be the tooltip for the `IsImmuneToCriticals` property.